addons/crm/views/crm_team_views.xml

```diff
@@ -83,12 +83,12 @@
                 <field name="arch" type="xml">
                     <xpath expr="//div[@name='options_active']" position="inside">
                         <div class="o_row">
-                            <span groups="crm.group_use_lead" class="o_row">
+                            <field name="use_opportunities"/>
+                            <label for="use_opportunities"/>
+                            <span groups="crm.group_use_lead" class="o_row" attrs="{'invisible': [('use_opportunities', '=', False)]}">
```

Comment:
Yes, leads is a sub-option of opportunities, (hides if unchecked)

though when saved it is not necessarily wanted for it to align with quotations ^^